pypika/clickhouse/dates_and_times.py

```diff
@@ -1,6 +1,102 @@
+import abc
+
 from pypika.terms import Function
 
 
 class FormatDateTime(Function):
     def __init__(self, term, alt, **kwargs):
         super().__init__('formatDateTime', term, alt, **kwargs)
+
+
+class ToYYYYMM(Function):
```

Comment:
Maybe it would be easier to implement these as one liners using the `CustomFunction` class? I just added some docs on it here https://pypika.readthedocs.io/en/latest/4_extending.html

Was planning on moving all of the other functions implemented in pypika to use that, since in most cases the only thing you need to define for functions are the SQL name and the list of args.

Reply:
ok - no problem ;)

thank you